jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/LogEventHandler.java

```diff
@@ -41,6 +41,13 @@ public void accept(LogEvent logEvent) {
         }
         break;
 
+      case PROGRESS:
+        // Progress messages only print if not printing progress in the footer.
```

Comment:
I think this might be hard to follow in the future if we haven't been looking at it. There's a lot of branching in a lot of places. 

Is it possible to structure this in a way that we just instantiate the right kind of progress object? (the cool one, and the boring one) or whatever?

Each of them just handles the progress correctly?

Reply:
Yep, I'll try splitting this class into two classes `RichLogEventHandler` and `PlainLogEventHandler` (or some other better naming).